client/query.go

```diff
@@ -126,19 +118,25 @@ func (c *QueryClient) queryRequest(queryResult interface{}) (err error) {
 }
 
 // generateQueryURL URL encodes the NRQL
-func (c *QueryClient) generateQueryURL(nrqlQuery string) error {
+func (c *QueryClient) generateQueryURL(nrqlQuery string) (string, error) {
 	if len(nrqlQuery) < minValidNRQLLength {
-		return fmt.Errorf("NRQL query is too short [%s]", nrqlQuery)
+		return "", fmt.Errorf("NRQL query is too short [%s]", nrqlQuery)
 	}
 
 	// Use a new set of Values to sanitize the query string
 	urlQuery := url.Values{}
 	urlQuery.Set("nrql", nrqlQuery)
-	c.URL.RawQuery = urlQuery.Encode() // Seems odd, but directly out of the docs for net/url
+	queryString := urlQuery.Encode() // Seems odd, but directly out of the docs for net/url
```

Comment:
Remove the comment, since we aren't writing directly into the URL struct now.

Reply:
Comment removed.